lib/js_fetcher.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
```

Comment:
As @lorawoodford mentioned in a comment in spec/models/searchgov_url_spec.rb, `js_fetcher` probably needs a unit test.